lib/db.js

```diff
@@ -740,16 +740,17 @@ Db.prototype.executeDbAdminCommand = function(selector, options, callback) {
  * @param {string} [options.name] Override the autogenerated index name (useful if the resulting name is larger than 128 bytes)
  * @param {object} [options.partialFilterExpression] Creates a partial index based on the given filter object (MongoDB 3.2 or higher)
  * @param {ClientSession} [options.session] optional session to use for this operation
+ * @param {(number|string)} [options.commitQuorum] (MongoDB 4.4. or higher) Specifies how many data-bearing members of a replica set, including the primary, must complete the index builds successfully before the primary marks the indexes as ready. This option accepts the same values for the "w" field in a write concern plus "votingMembers", which indicates all voting data-bearing nodes.
  * @param {Db~resultCallback} [callback] The command result callback
  * @return {Promise} returns Promise if no callback passed
  */
 Db.prototype.createIndex = function(name, fieldOrSpec, options, callback) {
   if (typeof options === 'function') (callback = options), (options = {});
   options = options ? Object.assign({}, options) : {};
 
-  const createIndexOperation = new CreateIndexOperation(this, name, fieldOrSpec, options);
+  const createIndexesOperation = new CreateIndexesOperation(this, name, fieldOrSpec, options);
```

Comment:
What's the idea here for the pluralization change and deletion of `create_index`?

Reply:
Everything already uses the same `createIndexes` command on the server, `CreateIndexOperation` previously just did some handling for the options/index spec/return value, and then internally constructed a `createIndexes` command to send to the server. Since they are sending the same command to the server, they can really share most of their code.

Now the `CreateIndexesOperation` detects internally if it was called in "singular mode" and does the special handling when executing the operation.